packages/fcl-wc/src/service.js

```diff
@@ -147,54 +155,31 @@ async function connectWc(onClose, {service, client, pairing}) {
   }
 }
 
-function makeWcServices(client) {
-  const pairings = client.pairing.getAll({active: true})
-  return [
-    {
-      definition: {
-        f_type: "Service",
-        f_vsn: "1.0.0",
-        type: "authn",
-        method: "WC/RPC",
-        uid: "wc#authn",
-        endpoint: "flow_authn",
-        optIn: false,
-        provider: {
-          address: "WalletConnect",
-          name: "WalletConnect",
-          uid: null,
-          icon: "https://avatars.githubusercontent.com/u/37784886",
-          description: "WalletConnect Generic Provider",
-          website: "https://walletconnect.com/",
-          color: null,
-          supportEmail: null,
-        },
-      },
-      strategy: makeServiceStrategy(client),
+const baseWalletConnectService = includeBaseWC => {
+  return {
+    f_type: "Service",
+    f_vsn: "1.0.0",
+    type: "authn",
+    method: "WC/RPC",
+    uid: "https://walletconnect.com",
+    endpoint: "flow_authn",
+    optIn: !includeBaseWC,
+    provider: {
+      address: null,
+      name: "WalletConnect",
+      icon: "https://avatars.githubusercontent.com/u/37784886",
+      description: "WalletConnect Base Service",
+      website: "https://walletconnect.com",
+      color: null,
+      supportEmail: null,
     },
-    ...pairings.map(pairing => {
-      return {
-        definition: {
-          f_type: "Service",
-          f_vsn: "1.0.0",
-          type: "authn",
-          method: "WC/RPC",
-          uid: pairing.topic,
-          endpoint: "flow_authn",
-          optIn: false,
-          provider: {
-            address: null,
-            name: pairing.peerMetadata.name,
-            uid: pairing.topic,
-            icon: pairing.peerMetadata.icons[0],
-            description: pairing.peerMetadata.description,
-            website: pairing.peerMetadata.url,
-            color: null,
-            supportEmail: null,
-          },
-        },
-        strategy: makeServiceStrategy(client),
-      }
-    }),
-  ]
+  }
+}
+
+async function makeWcServices({includeBaseWC, wallets}) {
+  const wcBaseService = baseWalletConnectService(includeBaseWC)
+  const flowWcWalletServices = await fetchFlowWallets()
+  const injectedWalletsServices =
```

Comment:
Is this to allow for local development on testnet? Should we also support emulator?

Reply:
Not sure which line you are referring to here. Emulator is tricky, as wallets would need to have those keys as part of the wallet. If you are referring to developing a wallet, we can look into this addition. Do you think it's needed out the gate?